src/core/lib/transport/transport.h

```diff
@@ -69,6 +69,13 @@
 
 #define GRPC_ARG_TRANSPORT "grpc.internal.transport"
 
+/** A comma separated list of supported transport protocols. If non-empty, allow
+ client and server to attempt to negotiate transport protocols.
+ NOTE: This is an experimental feature. It is not fully implemented and is not
+ currently functional.
+ TODO(gtcooke94) - update with specific details when implementing. */
+#define GRPC_ARG_TRANSPORT_PROTOCOLS "grpc.transport_protocols"
```

Comment:
nit: Needs to be changed to `grpc.internal.transport_protocols`.

Reply:
My bad, that change didn't make it up to origin